Source/Matter/EMTensor.hpp

```diff
@@ -19,22 +19,46 @@ template <class matter_t> class EMTensor
 {
   public:
     template <class data_t>
-    using Vars = typename MatterCCZ4<matter_t>::template Vars<data_t>;
+    using Vars = typename MatterCCZ4RHS<matter_t>::template Vars<data_t>;
+
+    /// derive record name
+    static inline const std::string name = "EMTensor";
+
+    /// Variable names
+    static inline amrex::Vector<std::string> var_names = {"Rho"};
```

Comment:
It's a lowercase $\rho$ I believe.
```suggestion
    static inline amrex::Vector<std::string> var_names = {"rho"};
```